MAINTAINERS.yml

```diff
@@ -1570,6 +1570,7 @@ ITE Platforms:
         - sjg20
     files:
         - boards/riscv/it8xxx2_evb/
+        - drivers/*/*/*it8xxx2*
```

Comment:
what are you trying to match here? if `drivers/sensor/ite_tach_it8xxx2/tach_ite_it8xxx2.c`, then you should adapt the other wildcard to match the files explicitly, otherwise you will end up with a conflict.

Reply:
Yes, I want to match the drivers/.../tach_ite_it8xxx2.c path.
Done, thanks.